### PR TITLE
CDA-1695 example in go

### PR DESCRIPTION
**Pull request by CodiePP**
example of submitting a query from a go program

### CODE REVIEW

**Review by yenat — APPROVED** [review]
:+1: 
:100:

File: examples/simple_query_in_go/main.go [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+	SlotNo      int64  `bigquery:"slot_no"`
+	BlockTime   bigquery.NullDateTime `bigquery:"block_time"`
+	BlockSize   int64  `bigquery:"block_size"`
+	TxCount     int64  `bigquery:"tx_count"`
+	SumTxFee    int64  `bigquery:"sum_tx_fee"`
+	ScriptCount int64  `bigquery:"script_count"`
+	ScriptSize  int64  `bigquery:"sum_script_size"`
+	PoolHash    string  `bigquery:"pool_hash"`
+}
+func block2String(b *Block) string {
```
Comment (thomaska) [inline examples/simple_query_in_go/main.go:69]: so the row is dynamically assigned to the type `Block`! 😮

**Review by thomaska — APPROVED** [review]
👍 💯

File: examples/simple_query_in_go/main.go (hunk 1 above)
Comment (CodiePP) [reply examples/simple_query_in_go/main.go:69]: > so the row is dynamically assigned to the type `Block`! 😮

the row is of type "Block" -> Line 85-86

**Pull request merged**